### PR TITLE
Change default donation website

### PR DESCRIPTION
**Pull request by Yardanico**

### CODE REVIEW

File: jekyll/donate.md [hunk 1]
```diff
@@ -12,16 +12,15 @@ css_class: donate
...
 <section class="background-faded call-to-action">
   <section class="content text-centered">
-    <h1>BountySource</h1>
+    <h1>Open Collective</h1>
     <div class="pure-g center">
       <div class="pure-u-1-2">
         <p>
-          Help us reach our next fundraiser goal by supporting our Salt
-          campaign.
+          Help us with the Nim development by contributing on Open Collective.
```
Comment (Yardanico) [inline jekyll/donate.md:19]: Not sure about the description here

**Review by dom96 — APPROVED** [review]
Seems good to me. Wow, bountysource has really gone downhill.

**Comment by narimiran** [comment]
> Seems good to me.

Can you add `hacktoberfest` topic to this repo, so when we merge this it counts as Hacktoberfest contribution for @Yardanico?

**Comment by Yardanico** [comment]
@narimiran actually you can do the same by labelling the PR "hacktoberfest-accepted" :)

**Comment by Yardanico** [comment]
Is the PR ready to be merged or should I change something?

**Pull request merged**